acceptance/analyzer_test.go

```diff
@@ -249,6 +277,8 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 
 		when("analyzed path is provided", func() {
 			it("writes analyzed.toml at the provided path", func() {
+				h.SkipIf(t, runtime.GOOS == "windows", "Not yet working on Windows")
```

Comment:
What do we need in order to make it run on Windows?

Reply:
Hmm... I recall there was some issue, but I want to say this could be removed now.